packages/react-i18n/src/currencyShortFormException.ts

```diff
@@ -0,0 +1,4 @@
+export const CurrencyShortFormException = {
```

Comment:
I had this thought in my head on where to put this, and the `constants` folder was an option. But because `currencyCode.ts` was in the `src` folder I felt `currencyShortFormException.ts` could live there too. Now on a second thought, I think the `constants` folder would be more appropriate for it.